packages/eas-cli/src/commands/release/view.ts

```diff
@@ -0,0 +1,171 @@
+import { getConfig } from '@expo/config';
+import { Command, flags } from '@oclif/command';
+import chalk from 'chalk';
+import Table from 'cli-table3';
+import gql from 'graphql-tag';
+import _ from 'lodash';
+
+import { graphqlClient, withErrorHandlingAsync } from '../../graphql/client';
+import { Update } from '../../graphql/generated';
+import log from '../../log';
+import { ensureProjectExistsAsync } from '../../project/ensureProjectExists';
+import { findProjectRootAsync, getProjectAccountNameAsync } from '../../project/projectUtils';
+import { promptAsync } from '../../prompts';
+
+type TruncatedUpdate = Pick<Update, 'updateGroup' | 'updateMessage' | 'createdAt' | 'actor'>;
+
+async function viewUpdateReleaseAsync({
+  appId,
+  releaseName,
+}: {
+  appId: string;
+  releaseName: string;
+}): Promise<{
+  id: string;
+  releaseName: string;
+  updates: TruncatedUpdate[];
+}> {
+  const data = await withErrorHandlingAsync(
+    graphqlClient
+      .mutation<
+        {
+          app: {
+            byId: {
+              updateReleaseByReleaseName: {
+                id: string;
+                releaseName: string;
+                updates: TruncatedUpdate[];
+              };
+            };
+          };
+        },
+        {
+          appId: string;
+          releaseName: string;
+        }
+      >(
+        gql`
+          query ViewRelease($appId: String!, $releaseName: String!) {
+            app {
+              byId(appId: $appId) {
+                updateReleaseByReleaseName(releaseName: $releaseName) {
+                  id
+                  releaseName
+                  updates(offset: 0, limit: 1000) {
```

Comment:
Should this be truncated to last 10? If so, should we paginate?

Reply:
Resolved in private discussion with @fson. We will return it all and allow `less` to handled it.